.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Tap '...'
+3. Scroll down to '...'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots or Video**
+If applicable, add screenshots or video to help explain your problem.
+
+**Technical info (please complete the following information):**
+ - OS: [e.g. iOS, iPadOS, macOS]
+ - App Version [e.g. 22]
```

Comment:
Can we rewrite this to: App Version [e.g. Nos Social 0.1.9 (239), you can find this information in TestFlight].

Reply:
@martindsq I updated this to include a better example of the app version. I chose not to include the part about TestFlight since I hope we can soon include the version in the app settings: https://github.com/orgs/planetary-social/projects/5/views/4?filterQuery=app+version&pane=issue&itemId=59358421

Reply:
Oh, and I expect this to mainly be used internally by people who already know how to find the app version.